css/christine-anthony.css

```diff
@@ -0,0 +1,30 @@
+h1 {
+  color: red;
+}
+
+#green {
+  color: green;
+}
+
+.bold-italic {
+  font-weight: bold;
+  font-style: italic;
+}
+
+li > blockquote {
+  text-decoration: underline;
+}
+
+body blockquote {
+  background-color: grey;
+}
+div {
```

Comment:
This `div` and it's styles were just for practice during our last class, and shouldn't be included in the final code for this project. Also, if you look in your markup, you'll notice there are no `div` elements at all - so this CSS rule isn't styling anything 😄